build.assets/makefiles/master/k8s-master/kube-scheduler.service

```diff
@@ -9,7 +9,11 @@ ExecStartPre=/bin/systemctl is-active kube-apiserver.service
 ExecStart=/usr/bin/kube-scheduler \
         --master=https://${KUBE_APISERVER}:6443 \
         --kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
+        --authentication-kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
```

Comment:
I think you're going to need to add some context for these changes. Why the additional kubeconfig's to the base configuration.